### PR TITLE
Improved rendering of blocks on Page records

### PR DESCRIPTION
**Pull request by matt-bernhardt**
### Why these changes are being introduced

We need to improve how a handful of Omeka's blocks render content. Most of the changes can be accomplished with CSS changes, but there is one instance where the markup must also change.

### How this addresses that need

This adds a series of SCSS modules to the theme (in a folder that matches where the markup is coming from for organizational clarity), and adds them to the compilation process. These modules generally correspond to the blocks that are being targeted, although since many blocks include the same `.resource` class I've included that as a separate module whose effects are felt more widely throughout the theme.

Beyond the CSS changes, this also copy-pastes one template that needs to be changed (for item showcase blocks), and adds a class indicating what size the contained images are. This allows us to select CSS rules to separate how large the showcase is rendered.

### Side effects of this change

The `.resource` class name is used pretty widely throughout Omeka, and there may be some impacts of this rule change on resource pages that are not intended. The next few prototyping tickets are going to address this side effect, as the resource pages themselves need to be styled as well.

### Relevant ticket(s)

* https://mitlibraries.atlassian.net/browse/post-84

Please note that the Jira ticket includes links to a Theme Playground which we've been building that shows how the various elements of the theme operate. One of the pages in the playground is specific to images; comparing that page on production to staging (where this branch is deployed) will show the impact of these changes.

### CODE REVIEW

**Review by jazairi — APPROVED** [review]
The only thing that gives me pause here is pulling in an upstream template, which could cause compatibility issues similar to what we see with `administrate` views in ETD. Here, as in ETD, it seems like a necessary evil, and the template in question doesn't seem so complex that it will be difficult to maintain or keep track of.

As to your question about whether an ADR is necessary, I think that's your call. My sense is that decisions that require unanimous buy-in should be ADR'd, whereas everything else can be documented as it passes code review.

My personal opinion is that this probably doesn't need an ADR, but it does feel like a good opportunity to document which template(s) are coming from upstream and should be verified during significant version upgrades. I don't think that needs to happen in this PR, so I'm approving it as-is; though, if you decide to add docs as a follow-on commit before merging, I'm happy to review them.

**Comment by matt-bernhardt** [comment]
Those are good points. I've added a ticket to our next sprint to try writing this level of documentation within the theme.

https://mitlibraries.atlassian.net/browse/POST-123

Thanks for your feedback on this point - I really appreciate it.

**Pull request merged**
